### ISSUE TITLE
Secure Storage is undefined? 

### ISSUE THREAD

**Issue by nickabond** [Crypho/cordova-plugin-secure-storage#14]
My code works great on iOS, with getting and setting items. However, I get SecureStorage is undefined when I run the same code on Android 4.4 with Crosswalk 15 and with Cordova 5.1.1?

Here's my code:

document.addEventListener("deviceready",  function() {
	var ss = new cordova.plugins.SecureStorage(
		function () {},
		function (error) { console.log('Error ' + error); },
		'YourChurch');
},false);

It set I needed to set a security PIN, so I figured that at least part of the library must have been loaded correctly?

**Comment by ggozad** [comment]
I will look into this next week, would it be possible to see if you can run the tests?

**Comment by demetris-manikas** [comment]
@nickabond 
I created an empty project and installed the plugin following the README
Then I run it on a freshly created AVD and everything went smoothly.

I also installed cordova-plugin-crosswalk-webview and again everything went fine on a Samsung 5.
(crosswalk refused to run on the emulator)

Make sure you fresh install the secure storage plugin from npm and try again.
Create a cordova project from scratch and see if the problem persists.

Bare in mind that at least on Android the secure storage is not available before the success function
of the new cordova.plugins.SecureStorage() has been called.

One thing I don't understand in your comment is how can it be that SecureStorage is undefined and to still get a message from the plugin.
